thrust/thrust/system/cuda/detail/copy_if.h

```diff
@@ -147,7 +146,7 @@ struct DispatchCopyIf
     // Return for empty problems
     if (num_items == 0)
     {
-      output_end = output;
+      output;
```

Comment:
Statement without side effects. Can be removed.

Reply:
🤦 obviously it was already too late